docs/user/polarization.rst

```diff
@@ -0,0 +1,395 @@
+Equations :eq:`full-s-off-zero-uncertainty` through :eq:`full-s-on-pi-uncertainty` also define a
+:math:`4 \times 4` matrix :math:`M_4` that can be inverted to obtain the uncertainties in the spin
+state SANS data sets, as shown in Equation :eq:`m4-matrix`. The x and y dimensions are no longer
+abbreviated below.
+
+.. math::
+    \begin{pmatrix}
+        S^{\uparrow\uparrow}(x, y, \lambda) \\
+        S^{\uparrow\downarrow}(x, y, \lambda) \\
+        S^{\downarrow\uparrow}(x, y, \lambda) \\
+        S^{\downarrow\downarrow}(x, y, \lambda)
+    \end{pmatrix} =
+    M_3^{-1}
+    \begin{pmatrix}
+        S^{00}(x, y, \lambda) \\
+        S^{10}(x, y, \lambda) \\
+        S^{0\pi}(x, y, \lambda) \\
+        S^{1\pi}(x, y, \lambda)
+    \end{pmatrix}
+    :label: m3-matrix
+
+.. math::
+    \begin{pmatrix}
+        \left( \delta S^{\uparrow\uparrow}(x, y, \lambda) \right)^2 \\
+        \left( \delta S^{\uparrow\downarrow}(x, y, \lambda) \right)^2 \\
+        \left( \delta S^{\downarrow\uparrow}(x, y, \lambda) \right)^2 \\
+        \left( \delta S^{\downarrow\downarrow}(x, y, \lambda) \right)^2
+    \end{pmatrix} =
+    M_4^{-1}
+    \begin{pmatrix}
+        \left( \delta S^{00}(x, y, \lambda) \right)^2 \\
+        \left( \delta S^{10}(x, y, \lambda) \right)^2 \\
+        \left( \delta S^{0\pi}(x, y, \lambda) \right)^2 \\
+        \left( \delta S^{1\pi}(x, y, \lambda) \right)^2
+    \end{pmatrix}
+    :label: m4-matrix
+
+Application of corrections to polarized beam data
+-------------------------------------------------
+
+In the kinematic limit where the scattering is weak compared to the intensity of the neutron
+beam, the transmission of the incident beam through the sample is not spin dependent. Thus,
+the transmission correction should be applied before the polarization correction. In most
+cases, sources of intrinsic and extrinsic background are spin independent. Subtraction of
+background for these cases should also be performed before application of the polarization
+corrections (otherwise a false spin dependent asymmetry may be introduced into the
+background). It is especially important to apply the polarization corrections to the wavelength
+dependence of the data before the data are mapped to reciprocal space.
+
+The possibility that the beam position changes with the states of the polarizer and analyzer is
+addressed by specifying different direct beam for each of the workspaces extracted from the event
+streams.
+
+Process Variables
+-----------------
+
+Process variables (PV) indicate the spin selector or flipper used and its state. Given a
+particular Polarizer, PV PolarizerState indicates whether the selected state is "up" or "down".
+PVs FlipperState and AnalyzerState play analogous roles for the flipper and analyzer, respectively.
+The following tables summarize the possible values of the polarization PVs.
+
++-----------------------+-------------------+
+| PV Polarizer          | PV PolarizerState |
```

Comment:
The `ON` of `OFF` values of `PolarizerState` actually refer to whether the polarizer's flipper is `ON` or `OFF`. Thus, there's no need for `PV Flipper` and `PV FlipperState`. Perhaps a more accurate name for `PolarizerState` is `PolarizerFlipper` ?